migrations/capcons/capabilities.go

```diff
@@ -60,6 +63,34 @@ func (c *AccountCapabilities) Record(
 	)
 }
 
+func (c *AccountCapabilities) ForEach(
```

Comment:
Maybe rename the function to indicate that it also has the side-effect of sorting, and maybe also add a comment to make it clear

Reply:
added in https://github.com/onflow/cadence/pull/3542/commits/f712fdea18aa174f9073a26156c7d101f891faba